android/java/AndroidManifest_user_permissions.xml

```diff
@@ -0,0 +1,2 @@
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE_SYSTEM_EXEMPTED" />
+<uses-permission android:name="android.permission.USE_EXACT_ALARM"/>
```

Comment:
i had to add above permission because of 
`Error: foregroundServiceType:systemExempted requires permission:[android.permission.FOREGROUND_SERVICE_SYSTEM_EXEMPTED] AND any permission in list:[android.permission.SCHEDULE_EXACT_ALARM, android.permission.USE_EXACT_ALARM] [ForegroundServicePermission]
    <service android:name="org.chromium.chrome.browser.vpn.wireguard.WireguardService" android:foregroundServiceType="systemExempted" android:exported="false">`

Reply:
We can use https://developer.android.com/about/versions/14/changes/fgs-types-required#special-use. In that case, we need to explain and it may delay the release as reviewers would be manually checking the details.